src/Test.UnitTests.Sarif.Driver/CoyoteTestOutputLogger.cs

```diff
@@ -126,5 +126,35 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        public void Write(LogSeverity severity, string format, object arg0)
+        {
+            throw new NotImplementedException();
```

Comment:
>NotImplementedException [](http://example.com/codeflow?start=22&length=23)

@suvamM We obviously will need your expertise to properly implement/enable this functionality (and remove stale stuff).

Reply:
@michaelcfanning Understood.